src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -199,6 +200,12 @@
   const ANONYMOUS_USER = 'AnonymousUser';
   const {anon_user_info_color_code, anon_user_info_emoji_code} =
     channelInfo?.rawJson?.channel || {};
+  const {anonProfileId} = useProfileHook();
+  const memberChat = channelInfo?.rawJson?.channel?.members?.find(
+    (item: any) => item.user_id !== anonProfileId
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This block of code retrieves the `anonProfileId` using the `useProfileHook` and finds a member chat where the user_id does not match the `anonProfileId`. This logic seems correct as it helps to distinguish between anonymous and non-anonymous users.
```diff
+   const {anonProfileId} = useProfileHook();
+   const memberChat = channelInfo?.rawJson?.channel?.members?.find(
+     (item: any) => item.user_id !== anonProfileId
+   );
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->